lib/mutant/meta/example.rb

```diff
@@ -1,7 +1,7 @@
 module Mutant
   module Meta
     class Example
-      include Adamantium, Concord::Public.new(:file, :node, :mutations)
+      include Adamantium, Anima.new(:file, :node, :node_type, :expected)
```

Comment:
Possibly OT, but for Anima assert that all the attributes are required? I was just wondering what would happen if extra keys were provided. Would it be possible (or useful) for Anima to create an `#initialize` method that uses the new hash-based args, eg:

```ruby
def initialize(file:, node:, node_type:, expected:)
  @file      = file
  @node      = node
  @node_type = node_type
  @expected  = expected
end
```